### PR TITLE
Taipan

### PR DESCRIPTION
**Pull request by AndreSonntag**
This merge request includes the microservice for the TAIPAN extractor. TAIPAN is a generic approach to the extraction of RDF triples from tables. Currently the microservice works well, but it needs the system Sparqlify to create RDF triples. TAIPAN is responsible for SML mapping, subject column identification and property recommendation. The real TAIPAN system is dockered.

Currently Sparqlify system is not implemented. This will be a task for the next semesters.

### CODE REVIEW

**Comment by prasanthhs** [comment]
@AndreSonntag  codacy issues. Can you please check?

File: taipan-ms/src/main/resources/application.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+### Spring properties
+# Service registers under this name
+spring:
+  application:
+    name: taipan-ms
+
+# Discovery Server Access
+eureka:
+  instance:
```
Comment (prasanthhs) [inline taipan-ms/src/main/resources/application.yml]: Uses name of Executor MS. Need to change to new?

File: taipan-ms/src/main/resources/application.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    metadataMap:
+      type: executer
+      friendlyname: Executer
+  client:
+    serviceUrl:
+      defaultZone: http://localhost:1111/eureka/
```
Comment (prasanthhs) [inline taipan-ms/src/main/resources/application.yml]: The other pull request requires url of Eureka server in different format with different port number.

File: taipan-ms/src/main/resources/application.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+# Service registers under this name
+spring:
+  application:
+    name: taipan-ms
+
+# Discovery Server Access
+eureka:
+  instance:
+    metadataMap:
+      type: executer
```
Comment (KHaack) [inline taipan-ms/src/main/resources/application.yml]: should use extractor type and an other friednly name :)

File: taipan-ms/src/main/resources/application.yml (hunk 2 above)
Comment (KHaack) [reply taipan-ms/src/main/resources/application.yml]: jep, but for this master state is everything ok

**Comment by guru5590** [comment]
Since the requested changes are fixed, I'll be merging this to master branch

**Pull request merged**